config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-presubmit-dra-capz.yaml

```diff
@@ -0,0 +1,220 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-azure:
+  - name: pull-cluster-api-provider-azure-load-test-dra-custom-builds
+    cluster: eks-prow-build-cluster
+    decorate: true
+    decoration_config:
+      timeout: 8h
+    path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    always_run: false
+    optional: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-community: "true"
+    branches:
+      - ^main$
+      - ^release-1.*
+    extra_refs:
+    - org: kubernetes-sigs
+      repo: cloud-provider-azure
+      base_ref: master
+      path_alias: sigs.k8s.io/cloud-provider-azure
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    - org: kubernetes
+      repo: perf-tests
+      base_ref: master
+      path_alias: k8s.io/perf-tests
+    spec:
+      serviceAccountName: azure
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250613-876fb90a97-master
+        command:
+        - runner.sh
+        - ./scripts/ci-entrypoint.sh
+        args:
+        - bash
+        - -c
+        - >-
+          sleep 300 &&
+          cd ${GOPATH}/src/k8s.io/perf-tests/ &&
+          ./run-e2e.sh cluster-loader2
+          --nodes=100 \
+          --provider=aks \
+          --testconfig=testing/load/config.yaml \
+          --testconfig=testing/huge-service/config.yaml \
+          --testconfig=testing/access-tokens/config.yaml \
+          --testoverrides=./testing/experiments/enable_restart_count_check.yaml \
+          --testoverrides=./testing/experiments/use_simple_latency_query.yaml \
+          --testoverrides=./testing/overrides/load_throughput.yaml \
+          --report-dir=${ARTIFACTS}
+          --v=2
```

Comment:
FYI @alaypatel07 we'd like to keep these new jobs in sync with the periodic ones, so if you could do your best to remember to make the same changes to these jobs too that would be great!

Reply:
Will do my best to remember.

In any case, thanks to @jackfrancis both of you should automatically be added as reviewers for any dra job change PRs, so we should be covered.